src/Cppyy.h

```diff
@@ -36,6 +36,14 @@ typedef unsigned long long PY_ULONG_LONG;
 typedef long double PY_LONG_DOUBLE;
 #endif
 
+namespace InterOp {
```

Comment:
@wlav, @sudo-panda, I am not particularly happy about introducing such an ODR violation but I was not sure how else we could do this.

I know we could use the factory approach where we call operator new on `TemplateArgInfo` but do we want to do that?

Reply:
`PY_LONG_DOUBLE` here isn't a problem b/c it will be defined from the configuration in `Python.h`. The problem of ODR breakage is in `cpp_cppyy.h` in the backend, where this originates and which does (explicitly) not depend on Python.

Not sure how much of a problem `long double` not existing on certain platforms is, but it's important in several cases that it doesn't default to `double`.

Reply:
I meant the copy of the TemplateArgInfo class.

Reply:
Oh. :) Similar applies then, but b/c of `pip` being unable to manage dependencies, one version of `cppyy` always explicitly selects one exact version of Cling (including patch number), so it's not a biggie.